app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file, new_file = (command[2:].split())
```

Comment:
There is no guarantee that the first two characters are 'cp'. So, please simply split the command and also check that the length of the split result equals 3. 
Additionally check that fist element is 'cp'